gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -22,6 +23,7 @@ import NxTablePaginationFilterExample from './NxTablePaginationFilterExample';
 
 const tableSimpleExampleCode = require('./NxTableSimpleExample?raw');
 const tableClickableExample = require('./NxTableClickableExample?raw');
+const tableClickableCustomExample = require('./NxTableClickableCustomExample?raw');
```

Comment:
This comment refers to line 239-241

The `clickAccessibleLabel` for `NxTable.Row` needs to be modified. It says "The accessible name to set on the click chevron", but in fact the accessible name could be for any other icon besides a chevron as well.

![image](https://user-images.githubusercontent.com/7110862/134004190-a1855310-2133-4eae-b411-226b1588f7ae.png)

Reply:
> This comment refers to line 239-241

The clickAccessibleLabel for NxTable.Row needs to be modified. It says "The accessible name to set on the click chevron", but in fact the accessible name could be for any other icon besides a chevron as well.

71f064f1